lib/pka_queue.c

```diff
@@ -819,7 +853,17 @@ int pka_queue_cmd_dequeue(pka_queue_t            *queue,
     // One might just read 32bits using a 'uint32_t *' but for calirity we
     // cast the addresse in the queue.
     cons_head     = queue->cons.head;
-    cmd_desc      = (pka_queue_cmd_desc_t *) &queue->mem[cons_head];
+    // Read the queue memory carefully, adjust and wrap around if needed.
+    if (cons_head + sizeof(pka_queue_cmd_desc_t) < queue->size)

```

Comment:
please use braces, to make the 'if' statement consistent with the 'else' statement